docs/using.rst

```diff
@@ -83,8 +82,7 @@ the circumstances in which each plugin can be used, and how to use it.
 Apache
 ------
 
-The Apache plugin currently requires an OS with augeas version 1.0; currently `it
-supports
+The Apache plugin requires Apache 2.4+ and Augeas 1.0+; currently `it supports
```

Comment:
This is a useful change, but while we're at it, I suggest that we get rid of the version references entirely! There is code in the Apache plugin to support 2.2 as well, so this information isn't current. We could just say "The Apache plugin currently supports...".